src/DevBetterWeb.Core/Entities/ArchiveVideo.cs

```diff
@@ -20,6 +21,7 @@ public class ArchiveVideo : BaseEntity, IAggregateRoot
   public int Views { get; set; } = 0;
 
   public List<Question> Questions { get; private set; } = new List<Question>();
+  public List<MemberFavoriteArchiveVideo> MemberFavorites { get; private set; } = new List<MemberFavoriteArchiveVideo>();
```

Comment:
You're probably following the bad examples already in this code, but generally in entities/aggregates you want to encapsulate collection properties, like this:

https://github.com/ardalis/CleanArchitecture/blob/main/src/Clean.Architecture.Core/ProjectAggregate/Project.cs#L12-L13